code/modules/surgery/organs/eyes.dm

```diff
@@ -5,15 +5,17 @@
 	organ_tag = "eyes"
 	parent_organ = "head"
 	slot = "eyes"
-	var/eye_colour = "#000000" // Should never be null
+	var/eye_color = "#000000" // Should never be null
```

Comment:
Still seems a bit odd that the wiki uses British English and the code uses American English, but having both being used at once can definitely get confusing.